scripts/datadog_wrapper

```diff
@@ -32,9 +32,30 @@ then
   then
     echo "Configuring for the .NET runtime!"
   fi
+  # Handle the CORECLR_PROFILER_PATH
+  DEFAULT_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
+  ARM64_PATH=/opt/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so
+  X64_PATH=/opt/datadog/linux-x64/Datadog.Trace.ClrProfiler.Native.so
+  MUSL_X64_PATH=/opt/datadog/linux-musl-x64/Datadog.Trace.ClrProfiler.Native.so
```

Comment:
I don't think there is a musl compliant runtime for .NET so I think it's safe to get rid of this one?

Reply:
Removed